src/tests/ftest/util/daos_utils.py

```diff
@@ -621,3 +636,116 @@ def pool_list_cont(self, pool, svc, sys_name=None, env=None):
+        self.sub_command_class.sub_command_class.svc.value = svc
+        self.env = env
+        return self._get_result()
+
+    def pool_get_attr(self, pool, attr, svc, env=None):
+        """Set pool attribute.
+
+        Args:
+            pool (String): Pool UUID.
+            attr (String): Pool UUID.
+            svc (String): Service replicas.
+            env (dict, optional): dictionary of environment variable names and
+                values (EnvironmentVariables). Defaults to None.
+
+        Returns:
+            CmdResult: Object that contains exit status, stdout, and other
+                information.
+
+        Raises:
+            CommandFailure: if the doas pool query command fails.
+        """
+        self.set_sub_command("pool")
+        self.sub_command_class.set_sub_command("get-attr")
+        self.sub_command_class.sub_command_class.pool.value = pool
+        self.sub_command_class.sub_command_class.attr.value = attr
+        self.sub_command_class.sub_command_class.svc.value = svc
+        self.env = env
+        return self._get_result()
+
+    def pool_list_attrs(self, pool, svc, env=None):
+        """List pool attributes.
+
+        Args:
+            pool (String): Pool UUID.
+            svc (String): Service replicas.
+            env (dict, optional): dictionary of environment variable names and
+                values (EnvironmentVariables). Defaults to None.
+
+        Returns:
+            CmdResult: Object that contains exit status, stdout, and other
+                information.
+
+        Raises:
+            CommandFailure: if the doas pool query command fails.
+        """
+        self.set_sub_command("pool")
+        self.sub_command_class.set_sub_command("list-attrs")
+        self.sub_command_class.sub_command_class.pool.value = pool
+        self.sub_command_class.sub_command_class.svc.value = svc
+        self.env = env
+        return self._get_result()
+
+def get_pool_uuid(query_stdout):
+    """Get pool UUID from pool query output.
+
+    Sample pool query output.
+    04/19-18:31:26.90 wolf-3 Pool 3e59b386-fda0-404e-af7e-3ff0a38d1f81,
+        ntarget=8, disabled=0
+    04/19-18:31:26.90 wolf-3 Pool space info:
+    04/19-18:31:26.90 wolf-3 - Target(VOS) count:8
+    04/19-18:31:26.90 wolf-3 - SCM:
+    04/19-18:31:26.90 wolf-3   Total size: 1000000000
+    04/19-18:31:26.90 wolf-3   Free: 999997440, min:124999680,
+        max:124999680, mean:124999680
+    04/19-18:31:26.90 wolf-3 - NVMe:
+    04/19-18:31:26.90 wolf-3   Total size: 0
+    04/19-18:31:26.90 wolf-3   Free: 0, min:0, max:0, mean:0
+    04/19-18:31:26.90 wolf-3 Rebuild idle, 0 objs, 0 recs
+
+    Args:
+        stdout (String): daos pool query output.
+
+    Returns:
+        String: Pool UUID.
+    """
+    uuid_pattern = re.compile('([0-9a-f-]{36})')
+    uuid_match = uuid_pattern.search(query_stdout)
+    return uuid_match.group()
+
+def get_sizes(query_stdout):
```

Comment:
This should also be removed and replaced by `sizes = daos_cmd.get_output("pool_query", pool=uuid, sys_name=name, svc=svc, sys=sys)` + adding `r"Total size: (\d+)"` to the [`METHOD_REGEX`](https://github.com/daos-stack/daos/blob/223d66a67cfb742e8cb745d03edad6381c22b32d/src/tests/ftest/util/daos_utils.py#L34) for the `"pool_query"` key.

Reply:
Done